plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/lookup/ToolItemLookup.java

```diff
@@ -87,7 +87,7 @@ public ToolItem getWorkbenchToolItem(Matcher<String> matcher) {
 		Shell swtShell = ShellLookup.getInstance().getWorkbenchShell();
 		String text = WidgetHandler.getInstance().getText(swtShell);
 		WidgetHandler.getInstance().setFocus(swtShell);
-		new WaitUntil(new ShellWithTextIsActive(text));
+		new WaitUntil(new ShellWithTextIsAvailable(text));
```

Comment:
I changed this one blindly, you're right, waiting for available shell is incorrect. However the whole part of getting shell, setting focus and waiting for it to be active is weird and then I noticed that this method should be deprecated since we dont differentiate between workbench tool item and other tool items and also this method isnt used anywhere anymore. So marked as deprecated :) btw new WorkbenchShell() cannot be used since we are on core layer, workbenchShell is from swt.